app/src/components/v-field-list/v-field-list.vue

```diff
@@ -1,5 +1,6 @@
 <template>
 	<v-list :mandatory="false" @toggle="loadFieldRelations($event.value)">
+		<slot name="append" />
```

Comment:
I believe this should be `prepend` rather than `append` 👍🏻

Reply:
totally didn't notice this 🤦‍♂️ fixed!